packages/terra-application-header/src/ApplicationHeader.scss

```diff
@@ -0,0 +1,26 @@
+:local {
+  .application-header {
+    background-color: rgb(36, 129, 202);;
+    // background-image: durp;
+    border: 0.07143rem solid rgba(255, 255, 255, 0.17);
+    height: 40px;
```

Comment:
Having a set height like this for the application header restricts it's responsiveness and flexibility. If you do need this, can you do a min-height instead?